src/components/SUE/report/SueReportSend.tsx

```diff
@@ -32,16 +35,22 @@ export const SueReportSend = ({
 }) => {
   const { sueConfig = {} } = useContext(ConfigurationsContext);
   const { sueReportScreen = {} } = sueConfig;
-  const { reportSendDone = {}, reportSendLoading = {}, showFeedbackSection } = sueReportScreen;
+  const {
+    reportSendDone = {},
+    reportSendLoading = {},
+    showFeedbackSection: feedbackSection
+  } = sueReportScreen;
   const { title: loadingTitle = '', subtitle: loadingSubtitle = '' } = reportSendLoading;
   const { title: doneTitle = '', subtitle: doneSubtitle = '' } = reportSendDone;
+  const [showFeedbackSection, setShowFeedbackSection] = useState(feedbackSection);
 
+  const keyboardHeight = useKeyboardHeight();
   const scrollViewRef = useRef(null);
 
   const { control, reset, handleSubmit } = useForm({
     defaultValues: {
       message: '',
-      ratingCount: 4
+      rating: 4
```

Comment:
4 should be default? let's have a global setting for this in `sueReportScreen` like `showFeedbackSection`!

Reply:
yes it makes sense. For this I made the `defaultRating` value adjustable from `globalSettings`. I gave the value 4 as default.